services/provider/server/server.go

```diff
@@ -146,7 +148,10 @@ func (s *OCSProviderServer) AcknowledgeOnboarding(ctx context.Context, req *pb.A
 		}
 		return nil, status.Errorf(codes.Internal, "Failed to update the storageConsumer. %v", err)
 	}
-
+	// create noobaa account CR
+	if err := s.consumerManager.CreateNoobaaAccount(ctx, req.StorageConsumerUUID); err != nil {
```

Comment:
there is a possibility (if client-op restarts) that this endpoint could be called more than once and we need to handle a resource is already created corresponding to same request.

Reply:
The name will be identical, so there is no way we will create 2 separate accounts.